cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
         return data
```

Comment:
This is a good use of the validate method to check the validity of the ticket before it is created. However, the ValidationError should be raised inside the validate_ticket method, not passed as an argument. This would make the code cleaner and more understandable.

Reply:
in the context of my task, everything is correct here, so I think it's better not to change anything related to this comment